tests/triggers/kit/small/upstream-load.yaml

```diff
@@ -44,18 +44,25 @@ spec:
     metadata:
       generateName: kit-small-upstream-load-
     spec:
+      timeout: "9h0m0s"
       pipelineRef:
-        name: loadtest
+        name: kitloadtest
```

Comment:
rename to `kitSmallUpstreamLoadTest`?

Reply:
Pipeline is going to be same for `small` , `medium` , `large` `XL` etc. We will be reusing it, that's one the main purposes of leveraging tekton pipelines  :D